File: project/src/app/models/shopping.ts
import { Data } from "@angular/router";

export class Shopping {
    shoppingCode:number;
    customerCode:number;
    customerName:string;
    orderDate:Data;
    totalAmount:number;
    remark:string;
    constructor(shoppingCode:number,customerCode:number,customerName:string,orderDate:Data, totalAmount:number,remark:string)
    {
this.shoppingCode=shoppingCode;
this.customerCode=customerCode;
this.customerName=customerName;
this.orderDate=orderDate;
this.totalAmount=totalAmount;
this.remark=remark

    }
}

File: WebApi&angular/Dal/models/Shopping.cs
using System;
using System.Collections.Generic;

namespace Dal.models;

public partial class Shopping
{
    public int ShoppingCode { get; set; }

    public int CustomerCode { get; set; }

    public DateOnly OrderDate { get; set; }

    public string? Remark { get; set; }

    public decimal? TotalAmount { get; set; }

    public virtual Customer CustomerCodeNavigation { get; set; } = null!;

    public virtual ICollection<ShoppingDetail> ShoppingDetails { get; set; } = new List<ShoppingDetail>();
}
